### PR TITLE
feat: Change logs to JSON format

### PR DESCRIPTION
**Pull request by karolsojko**
- Added Lograge for JSON formatting of logs - if we keep logs in multiline it is hard to parse them in infra toolings like Sentry/CloudWatch/Datadog - additionally one event generates a couple of lines of logs which is not efficient. (source: https://www.datadoghq.com/blog/monitoring-rails-with-datadog/#fine-tuning-your-rails-logs)
- Fixed missing 404 page - previously a missing page/resource was treated as an Error with full stack trace from Rails (vide: https://github.com/roidrage/lograge/issues/146)

IP is still not logged. Adding screenshots to compare how logs look like with `RAILS_ENV` set to `development` and `production`

Zrzut ekranu 2020-07-27 o 11 40 18
Zrzut ekranu 2020-07-27 o 11 39 23

### CODE REVIEW

**Comment by mobitar** [comment]
Timestamps appear to be missing when running locally..you can't seem to tell when a request occurred.

**Comment by karolsojko** [comment]
> Timestamps appear to be missing when running locally..you can't seem to tell when a request occurred.

added :)

**Comment by mobitar** [comment]
Needs STDOUT check? https://github.com/standardnotes/syncing-server/blob/master/config/environments/raspberry_pi.rb#L14

**Comment by johnny243** [comment]
Should we format `ActiveRecord` output in development too?

**Comment by karolsojko** [comment]
> Should we format `ActiveRecord` output in development?

I think it makes sense to get rid of multiline formatting everywhere
